_posts/know-your-bounds.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+author: Mauro Servienti
+title: "Know your bounds. Infinite scalability doesn't exist"
```

Comment:
"Know your limits" might be more appropriate

Reply:
```suggestion
title: "Know your limits. Infinite scalability doesn't exist"
```